packages/core/src/DdSdkReactNative.tsx

```diff
@@ -133,7 +136,9 @@ export class DdSdkReactNative {
                         .version,
                     // eslint-disable-next-line global-require, @typescript-eslint/no-var-requires
                     reactVersion: require('react/package.json').version
-                }
+                },
+                configuration.bundleLogsWithRum,
+                configuration.bundleLogsWithTraces
```

Comment:
Do we also need to pass the data here since we seem to get it from the `additionalConfig` field anyway?